spec/spec_helper.rb

```diff
@@ -22,6 +22,10 @@ class Post < ActiveRecord::Base; end
   ActiveRecord::Base.logger = Logger.new(ENV['VERBOSE'] ? $stdout : nil)
   ActiveRecord::Migration.verbose = ENV.fetch('VERBOSE', nil)
 
+  # Configure time_zone like it would be in rails applications
+  ActiveRecord::Base.time_zone_aware_attributes = true
+  Time.zone = 'Berlin'
```

Comment:
I would rather mock the response values or change the setup only in the tests where we need it rather than changing the configuration for the whole test suite. Something like the code below would be enough probably.

```rb
  # This can be different for different contexts
  before do
    ActiveRecord::Base.time_zone_aware_attributes = true
    Time.zone = 'Berlin'
  end

  after do
    # what you did in the file above
  end
```
It looks like stubbing `Time.zone` can be problematic but I found this: https://stackoverflow.com/a/26741932